File: pepper-farm/backend/Models/Role.cs
namespace PepperFarm.Api.Models;

public class Role
{
    public int RoleId { get; set; }
    public string RoleName { get; set; } = string.Empty;
    public string? RoleDescription { get; set; }
    public bool IsActive { get; set; } = true;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: pepper-farm/backend/models/role.py
from sqlalchemy import Column, Integer, String, Boolean, DateTime
from sqlalchemy.sql import func
from database import Base

class Role(Base):
    __tablename__ = "Roles"

    RoleId          = Column(Integer, primary_key=True, autoincrement=True)
    RoleName        = Column(String(50), nullable=False, unique=True)
    RoleDescription = Column(String(255), nullable=True)
    IsActive        = Column(Boolean, nullable=False, default=True)
    CreatedAt       = Column(DateTime, nullable=False, server_default=func.sysutcdatetime())
